common/app/renderers/DotcomRenderingService.scala

```diff
@@ -137,6 +135,27 @@ class DotcomRenderingService extends GuLogging with ResultWithPreconnectPreload
     post(ws, json, Configuration.rendering.baseURL + "/Article", page)
   }
 
+  def getBlocks(
+      ws: WSClient,
+      page: LiveBlogPage,
+      blocks: Seq[Block],
+  )(implicit request: RequestHeader): Future[String] = {
+
+    val dataModel = DotcomBlocksRenderingDataModel(page, request, blocks)
+    val json = DotcomBlocksRenderingDataModel.toJson(dataModel)
+
+    ws.url(Configuration.rendering.baseURL + "/Blocks")
+      .withRequestTimeout(Configuration.rendering.timeout)
+      .addHttpHeaders("Content-Type" -> "application/json")
+      .post(json)
+      .map(response => {
+        if (response.status == 200)
+          response.body
+        else
+          throw DCRRenderingException(s"Request to DCR failed: status ${response.status}, body: ${response.body}")
```

Comment:
we should return a failed future and deal with that in the normal flow, in general I think it's good to avoid throwing or returning in Scala as it makes the code more difficult to follow. So maybe something like this
```
          .flatMap(response => {
        if (response.status == 200)
          Future.successful(response.body)
        else
          Future.failed(DCRRenderingException(s"Request to DCR failed: status ${response.status}, body: ${response.body}"))
      }
```